src/arithmetic.c

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file arithmetic.c
```

Comment:
Good, now put at top of every file (should be an automatic pre fill with a vscode extension)